src/content/News.md

```diff
@@ -1,3 +1,20 @@
+# Partnership with BridgingApps 
+BridgingApps is a website of Easter Seals Greater Houston. The Easter Seals network is the largest provider of disability services in North America, outside of the government. Created in 2010, BridgingApps aims to help parents of children with disabilities shortcut the process of finding appropriate apps to help their children learn, progress in therapy, and enhance their lives.  With over 3000 apps now represented on the database, Bridging Apps serves children, adults, caregivers, veterans, among other populations. Check out more about the collaboration here: https://www.bridgingapps.org/2020/10/bridgingapps-collaborates-with-harvard-digital-psychiatry-team-to-highlight-new-tool-for-mental-health-apps/
+
+# FTC Presentation
+The team at the Division of Digital Psychiatry presented at the Federal Trade Commission's Privacy Con in July of 2020 alongside other leading researchers of health privacy and health technology. For more information, as well as a recording of the presentation, check out this link: https://www.ftc.gov/news-events/events-calendar/privacycon-2020 and this link: https://www.natlawreview.com/article/ftc-privacycon-2020-health-apps-area-ftc-focus-going-forward. 
+
+# NAMI's Ask the Expert
+The National Alliance on Mental Illness (NAMI) is the largest grassroots mental health organization in the United States. In May of 2020, Dr. John Torous of the Division of Digital Psychiatry led an "Ask the Expert" session about how to choose a mental health app, introducing the app evaluation model and database to hundreds of webinar attendees. The session can be viewed at this link: https://nami.org/Blogs/NAMI-s-Ask-the-Expert/2020/NAMI-s-Ask-the-Expert-Webinar-What-to-Know-Before-Using-a-Mental-Health-App.
+
+# MIND Featured by STAT News
+The popular health outlet STAT news featured our database in July of 2020: https://www.statnews.com/2020/07/10/new-tool-sort-mental-health-app/. 
+
+# MIND Featured on Psychology Today
+The development and maintenance of MIND would be impossible without the dedicated team of app raters -- a group of students, researchers, professionals, and app users hailing from all over the world. Check out the fantastic work of a few of our app raters on these Psychology Today pieces: https://www.psychologytoday.com/us/blog/digital-mental-health/202009/digital-tools-depression and https://www.psychologytoday.com/us/blog/digital-mental-health/202007/connecting-teens-mental-health-tools.
+
+# Papers and More 
+The creation of the Mhealth Index and Navigation Database has also been published in academic literature. For more about the database creation, check out "Actionable Health App Evaluation" in NPJ Digital Medicine: https://www.nature.com/articles/s41746-020-00312-4. For more about how the MIND framework is flexible enough to serve broad use cases, including emerging COVID-19 apps, check out this Future Internet piece: https://www.mdpi.com/1999-5903/12/9/153. 
```

Comment:
Same here, you can cleanup the link addresses if you want.  See note above for example.